### PR TITLE
[iCubGenova09] Robots configuration to be used on the robot

### PR DESCRIPTION
**Pull request by GiulioRomualdi**
This PR merge #22 and #21 in order to simplify the usage on the robot. 

cc @S-Dafarra @isorrentino

### CODE REVIEW

**Comment by isorrentino** [comment]
Hi @GiulioRomualdi, I just pushed an additional commit to https://github.com/ami-iit/robots-configuration/pull/22. Can you align the PR?

**Comment by GiulioRomualdi** [comment]
Yes I will do it :)

**Comment by S-Dafarra** [comment]
Nitpicks: some files do not have the terminal line

File: iCubGenova09/estimators/wholebodydynamics.xml [hunk 1]
```diff
@@ -128,7 +159,7 @@
         <action phase="startup" level="15" type="attach">
             <paramlist name="networks">
                 <!-- motorcontrol and virtual torque sensors -->
-		<elem name="head-j0">head-eb20-j0_1-mc</elem>
+		        <elem name="head-j0">head-eb20-j0_1-mc</elem>
```
Comment (S-Dafarra) [inline iCubGenova09/estimators/wholebodydynamics.xml:162]: Unwanted indentation I guess

File: iCubGenova09/estimators/wholebodydynamics.xml [hunk 2]
```diff
@@ -145,14 +176,16 @@
...
+                <elem name="secondaryCalibrationLeftArm">secondaryCalibrationLeftArm</elem>
+                <elem name="secondaryCalibrationRightArm">secondaryCalibrationRightArm</elem>
+          		<!--elem name="l_arm_ft_sensor">left_arm-eb1-j0_1-strain</elem-->
+                <!--elem name="r_arm_ft_sensor">right_arm-eb3-j0_1-strain</elem-->
                 <elem name="l_foot_ft_sensors">left_leg-eb8-j3_5-strain</elem> <!-- left_rear, left_front -->
-		<elem name="l_leg_ft_sensor">left_leg-eb7-j0_2-strain</elem>    <!-- left_upper_leg -->
+		        <elem name="l_leg_ft_sensor">left_leg-eb7-j0_2-strain</elem>    <!-- left_upper_leg -->
                 <elem name="r_foot_ft_sensors">right_leg-eb12-j3_5-strain</elem> <!-- right_rear, right_front -->
-		<elem name="r_leg_ft_sensor">right_leg-eb11-j0_2-strain</elem>    <!-- right_upper_leg -->
+		        <elem name="r_leg_ft_sensor">right_leg-eb11-j0_2-strain</elem>    <!-- right_upper_leg -->
```
Comment (S-Dafarra) [inline iCubGenova09/estimators/wholebodydynamics.xml:188]: Also here the indentation seems odd

**Pull request closed without merging**